Week-3/Exercise(3.1)/memoize.js

```diff
@@ -0,0 +1,31 @@
+// Timer Function
+const timer = (fn) => {
+  console.time();
+  console.log(fn());
+  console.timeEnd();
+};
+
+// Main Function
```

Comment:
The below add function can only accepts 2 variables hence we can only add 2 numbers. But as per Guideline #4 -> The "function should be able to take n number of argument" .

Reply:
Done - Implemented for n number of arguments